### PR TITLE
Backend dev

### PR DESCRIPTION
**Pull request by anthony-yeo**
Test cases should be finished, so backend should be done for the most part.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+DB_PASSWORD=3Ik9fSJZ9o7j9DYJ
```
Comment (Aorta3698) [inline .env]: oh no.

**Review by Aorta3698 — CHANGES_REQUESTED** [review]
For whatever reason, the `.env` file got added into the repo. It needs to be removed. 

Please verify that the `.gitignore` file is filtering properly.

File: .env (hunk 1 above)
Comment (anthony-yeo) [reply .env]: Good catch, don't know why this happened, it has been removed

**Review by Aorta3698 — CHANGES_REQUESTED** [review]
Good work @anthony-yeo . Could you also support get requests given an event id?

For example, as of now a get request for `http://localhost:5000/events/63f93fa714c3a4d743e9168b`  is not valid. 

This will come in handy when a user tries to view an event that they've saved previously. I am trying to link the home page to the backend rn.

**Review by Aorta3698 — CHANGES_REQUESTED** [review]
Just hit a brick wall when linking - the login post call is returning `true` or `false`, but I think it will make more sense for it to return the json data for the user when the email & password match, and `false` otherwise. 

It is so that the frontend can display user info after a successful login.

**Comment by anthony-yeo** [comment]
@anarchaworld Validateusers now return the user info instead of a boolean. Also refer to the getEvents function in eventServices. There is an example of how to provide a query with the event id.

**Review by Aorta3698 — APPROVED** [review]
Looks Good! Thanks @anthony-yeo

**Review by RandallCaler — APPROVED** [review]
Looks good! Don't see any glaring issues, and it seems to follow convention.

**Pull request merged**